pygfx/renderers/wgpu/_renderer.py

```diff
@@ -266,6 +287,21 @@ def blend_mode(self, value):
         if isinstance(self._target, wgpu.gui.WgpuCanvasBase):
             self._target.request_draw()
 
+    @property
+    def sort_objects(self):
+        """Whether to sort world objects before rendering.
+
+        * ``True``: the render order is defined by 1) the object's distance to the
+          camera; 2) the object's ``render_order`` property; 3) the position object
+          in the scene graph (based on a depth-first search).
+        * ``False``: don't sort, the render order is defined by the scene graph alone.
+        """
+        return self._sort_objects
```

Comment:
* Could also allow setting a function object (in this PR or later).
* Could also allow `None` to sort depending on whether the `blend_mode` is order independent. I think we should not do this, because when a user changes the bend mode, the user may just as well edit the `sort_objects` as well.